simtools/applications/generate_default_metadata_file.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    Generate a default simtools metadata file from a json schema.
```

Comment:
Not sure what to add. It generates a metadata file from a json schema.

I don't like adding the current dict the application will return, this might be outdated very soon.

I've added an example.